raw_data/preprocessed/africa/seychelles_english.txt

```diff
@@ -0,0 +1,91 @@
+
+(26) The Minister may by notice in the Gazette, appoint or recognize such person or persons as the Minister thinks fit and having the prescribed qualifications to be Government Analysts for the purpose of analysis of samples of air, water, soil or other substance sent for analysis to any environmental laboratory established or recognized under the Act.
+
+(27) Any document purporting to be a report signed by the Government analyst may without proof of signature, be used as evidence of the facts stated therein in any proceeding under this Act. 
+
+PART VI - OFFENCES, PENALTIES AND PROCEDURES
+
+(29) [1] Subject to subsection [2], a person who is guilty of an offence under section 12[4][a] or [b] is liable on conviction to a fine of R5.000. [2] Where on any occasion, an authorised officer finds a person who the officer has a reason to believe has on that occasion committed an offence under section 12[4][a] or [b], the officer may give that person a notice offering the person the opportunity of discharging any liability to conviction for that offence by payment of a fixed penalty. [3] Where a person is given a notice under this section in respect of an offence: [a] no proceedings shall be instituted for that offence before the expiration of fourteen days following the date of the notice; [b] the person shall not be convicted of that offence if the person pays the fixed penalty before the expiration of that period. [4] A notice under this section shall give such particulars of the circumstances alleged to constitute the offence as are necessary for giving reasonable information of the offence and shall state: [a] the period during which, by virtue of subsection [3] proceedings will not be taken for the offence; [b] the amount of fixed penalty; and [c] the person to whom and the address at which the fixed penalty may be made or paid. [5] A fixed penalty specified in a notice under subsection [4] may be paid in cash to the person specified in the notice or by cheque enclosed in a letter addressed to that person at the address specified in the notice. [6] Where a letter is sent in accordance with subsection [5], the payment shall be regarded as having been made at the time at which that letter would be delivered in the ordinary course of post. [7] The form of notices under this section shall be such as may be prescribed under the regulations. [8] The fixed penalty payable in pursuance of a notice under this section shall, subject to subsection [9], be R100. [9] The Minister may by order substitute a different amount for the time being specified as the amount of the fixed penalty in subsection [8]. [10] In any proceedings a certificate which purports to be signed by or on behalf of the person receiving the fixed penalty stating that the payment of a fixed penalty was or was not received by a date specified in the certificate, shall, without proof of signature, be evidence of the facts stated therein.
```

Comment:
Final edit: `5.000` still needs to be replaced with `5,000`.

Reply:
Sorry about that, missed that one.